examples/nlp/text_classification/text_classification_with_bert.py

```diff
@@ -17,14 +17,17 @@
 import pytorch_lightning as pl
 from omegaconf import DictConfig
 
-from nemo.collections.nlp.models.text_classification_model import TextClassificationModel
+from nemo.collections.nlp.models.text_classification.text_classification_model import TextClassificationModel
```

Comment:
in the example we should do:
```
from nemo.collections.nlp.models import TextClassificationModel
```
please use ```nemo/collections/nlp/models/__init__ ```  to expose it

Reply:
done.

Reply:
done.